api/types/src/headers.rs

```diff
@@ -17,3 +17,5 @@ pub const X_APTOS_OLDEST_BLOCK_HEIGHT: &str = "X-Aptos-Oldest-Block-Height";
 pub const X_APTOS_LEDGER_TIMESTAMP: &str = "X-Aptos-Ledger-TimestampUsec";
 /// Cursor used for pagination.
 pub const X_APTOS_CURSOR: &str = "X-Aptos-Cursor";
+/// Provided by the client to identify what client it is.
+pub const X_APTOS_CLIENT: &str = "x-aptos-client";
```

Comment:
This is lowercase on purpose, using uppercase letters in a header name is actually not standard compliant. We should change the other ones too at some point.